tools/src/scie_pants/install_pants.py

```diff
@@ -116,8 +117,13 @@ def main() -> NoReturn:
     )
     info(f"New virtual environment successfully created at {venv_dir}.")
 
+    # Added in https://github.com/pantsbuild/pants/commit/558d843549204bbe49c351d00cdf23402da262c1
+    native_client_binaries = glob(str(venv_dir / "lib/python*/site-packages/pants/bin/native_client"))
+    maybe_native_client_binary = native_client_binaries[0] if len(native_client_binaries) == 1 else venv_dir / "bin" / "pants"
```

Comment:
Hrm, it looks like CI is running isort & fmt but not in check mode - this is >100 cols: https://github.com/pantsbuild/scie-pants/actions/runs/4941570260/jobs/8834300615?pr=172#step:10:400

Reply:
https://github.com/pantsbuild/scie-pants/pull/176